### PR TITLE
Update conditionals.mdx

### PR DESCRIPTION
**Pull request by reddynn**
this change just conditional error in the document

### CODE REVIEW

**Comment by radeksimko** [comment]
Hi @reddynn Can you please clarify what is wrong with the conditional currently? As far as I can tell it is just an example, so any operator is relevant there.

**Comment by reddynn** [comment]
in the explanation  mentioned as `If var.a is an empty string then the result is "default-a"` ,according to explanation there should not be "not equal operator" in condition

**Comment by radeksimko** [comment]
I see - good catch.

In that case however it should be `==` (double equal sign) as per https://developer.hashicorp.com/terraform/language/expressions/operators#equality-operators

File: website/docs/language/expressions/conditionals.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ A common use of conditional expressions is to define defaults to replace
 invalid values:
 
 ```hcl
-var.a != "" ? var.a : "default-a"
+var.a = "" ? var.a : "default-a"
```
Comment (radeksimko) [inline website/docs/language/expressions/conditionals.mdx:30]:
```suggestion
var.a == "" ? var.a : "default-a"
```

**Comment by reddynn** [comment]
updated the suggestion

**Comment by radeksimko** [comment]
Actually I stand corrected - the comment matches the behaviour in the original example. The conditional expression works like this:

```
CONDITION ? TRUE : FALSE
```

I think we could still remove the `!=` like proposed because negatives tend to be more confusing but now the comment is incorrect, so we either need to swap the two arguments or update the comment.

**Comment by reddynn** [comment]
its better to update arguments, so readers can avoid confusion, i updated the arguments please check

**Review by radeksimko — APPROVED** [review]

**Pull request merged**